configurations/longevity-100gb-2h-rackaware-validation.yaml

```diff
@@ -0,0 +1,13 @@
+test_duration: 180
+stress_duration: 120
```

Comment:
any specific reason we shorten the case to half of the original ?

Reply:
Do we need the long test here? there are 7 nemeses triggered during 2 hours. Do you think we need more?